generators/client/templates/src/main/webapp/app/services/profiles/_active-profiles.service.js

```diff
@@ -0,0 +1,57 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('<%=angularAppName%>')
+        .factory('ActiveProfiles', ActiveProfiles);
+
+    ActiveProfiles.$inject = ['$q', '$resource'];
+
+    function ActiveProfiles($q, $resource) {
+
+        var _activeProfiles;
+
+        var service = {
+            activeProfiles : activeProfiles
```

Comment:
I think can be seperate methods
On 27 Apr 2016 16:13, "Davy De Waele" <notifications@github.com> wrote:

> In
> generators/client/templates/src/main/webapp/app/services/profiles/_active-profiles.service.js
> <https://github.com/jhipster/generator-jhipster/pull/3474#discussion_r61219549>
> :
>
> > @@ -0,0 +1,57 @@
> > +(function() {
> > +    'use strict';
> > +
> > +    angular
> > +        .module('<%=angularAppName%>')
> > +        .factory('ActiveProfiles', ActiveProfiles);
> > +
> > +    ActiveProfiles.$inject = ['$q', '$resource'];
> > +
> > +    function ActiveProfiles($q, $resource) {
> > +
> > +        var _activeProfiles;
> > +
> > +        var service = {
> > +            activeProfiles : activeProfiles
>
> ok for ProfileService.
>
> Instead of exposing activeProfiles, should it return 1 response object
> containing
>
>    - activeProfiles
>    - ribbonEnv (will rename this to ribbonProfile I think)
>    - inProduction
>
> or do you want individual methods for that on the ProfileService ?
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/jhipster/generator-jhipster/pull/3474/files/04ad2b19006023e2670f0fc707e8d187936ce9aa#r61219549>
>